### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by NirobNabil**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# https://git-scm.com/docs/gitignore
```
Comment (azmainamin) [inline .gitignore:1]: Good work with the gitignore files!

File: front-end-master/common_files/jquery-ui-1.12.1/desktop.ini [hunk 2]
```diff
@@ -2,4 +2,3 @@
 InfoTip=This folder is shared online.
 IconFile=C:\Program Files\Google\Drive\googledrivesync.exe
```
Comment (azmainamin) [inline front-end-master/common_files/jquery-ui-1.12.1/desktop.ini]: Should the desktop.ini file be committed? Even if yes, the hardcoded path might pose a problem when we deploy to firebase as the path won't match

File: front-end-master/landing_page/index.css [hunk 3]
```diff
@@ -111,21 +112,15 @@ a, a:hover{
...
-		padding: 0;
-		/position: absolute;
-		z-index: -1;
-		/width: 200%;
-		/left: 15%;
-	}
+      z-index: -1;
+      left: 50%;
+      position: absolute;
+      min-height: 100%;
```
Comment (azmainamin) [inline front-end-master/landing_page/index.css:52]: Good work using relative % instead of px

File: front-end-master/landing_page/index.html [hunk 4]
```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
```
Comment (azmainamin) [inline front-end-master/landing_page/index.html:6]: We probably should add more <meta/> tags to enhance SEO. Don't worry about it now. We will create new work action items to take care of it later.

https://www.wordstream.com/meta-tags
https://moz.com/blog/seo-meta-tags

File: front-end-master/landing_page/index.html [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+	<link rel="stylesheet" href="./index.css">
+	<link href="https://fonts.googleapis.com/css?family=Pacifico|Patua+One|Roboto|Roboto+Condensed|Lato|Raleway" rel="stylesheet">
+    <link rel="stylesheet" media="screen and (max-device-width: 750px)" href="mobile.css" />
+
+
+
+	<!-- jquery -->
+	<script src="../common_files/js/jquery-3.1.1.js"></script>
+	<script src="index.js"></script>
+	<title>TheRSquaredProject!</title>
```
Comment (azmainamin) [inline front-end-master/landing_page/index.html]: Change this to The R-Squared Project

File: front-end-master/landing_page/index.html [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+			<div class="nav-links">
+				<a href="#" class="nav-item">Home</a>
+				<a href="#" class="nav-item">Projects</a>
+				<a href="#" class="nav-item">Acievements</a>
+				<a href="#" class="nav-item">About Us</a>
+				<button type="button" class="btn">Contact Us</button>
+			</div>
+		</nav>
+		<div class="land-content container col-8 offset-2">
+			<span class="land-text headline">Understanding bangladesh through data</span>
```
Comment (azmainamin) [inline front-end-master/landing_page/index.html]: Sp: *Bangladesh

File: front-end-master/landing_page/index.html [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+
+	<div class="scrollify container-fluid land">
+		<nav class="navbar nav-container">
+			<div class="nav-logo">
+				<span class="logo-text navbar-brand">R-Squared</span>
+			</div>
+			<div class="nav-links">
+				<a href="#" class="nav-item">Home</a>
+				<a href="#" class="nav-item">Projects</a>
+				<a href="#" class="nav-item">Acievements</a>
```
Comment (azmainamin) [inline front-end-master/landing_page/index.html]: *Sp: Achievements

**Review by azmainamin — CHANGES_REQUESTED** [review]
resolve the issues and it is good to go

File: front-end-master/landing_page/index.html (hunk 7 above)
Comment (NirobNabil) [reply front-end-master/landing_page/index.html]: Typing mistake :3

File: front-end-master/landing_page/index.css (hunk 3 above)
Comment (NirobNabil) [reply front-end-master/landing_page/index.css:52]: Thanks

File: front-end-master/common_files/jquery-ui-1.12.1/desktop.ini (hunk 2 above)
Comment (NirobNabil) [reply front-end-master/common_files/jquery-ui-1.12.1/desktop.ini]: Sorry i didn't notice that file.

**Review by azmainamin — APPROVED** [review]
Can you add desktop.ini file to gitignore?

**Pull request merged**